homeassistant/components/youless/sensor.py

```diff
@@ -40,9 +41,11 @@ async def async_setup_entry(
     )
 
 
-class YoulessBaseSensor(CoordinatorEntity, Entity):
+class YoulessBaseSensor(CoordinatorEntity, SensorEntity):
     """The base sensor for Youless."""
 
+    _attr_last_reset = dt.utc_from_timestamp(0)
```

Comment:
This is not a good default as this only applies to metered entities. This is now also applied to the power sensor, which doesn't reset.